file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,322 @@
+      "! <p class="shorttext">Name</p>
+      "! This is the name of the modelling context
+      name        TYPE c LENGTH 30,
+      "! <p class="shorttext">UUID</p>
+      "! This is the modelling context UUID
+      guid        TYPE c LENGTH 36,
+      "! <p class="shorttext">Description</p>
+      "! This is the description
+      description TYPE c LENGTH 100,
+    END OF ty_modelling_context,
+
+    "! <p class="shorttext">Modeling Context Filters</p>
+    "! These are the modeling context filters
+    BEGIN OF ty_modelling_context_filters,
+      "! <p class="shorttext">MC Name</p>
+      "! This is the modelling context name
+      mc_name          TYPE c LENGTH 30,
+      "! <p class="shorttext">Name</p>
+      "! This is the name
+      name             TYPE c LENGTH 30,
+      "! <p class="shorttext">Index</p>
+      "! This is the filter index
+      filter_index     TYPE i,
+      "! <p class="shorttext">UUID</p>
+      "! This is the modelling context UUID
+      modeling_context TYPE c LENGTH 36,
+      "! <p class="shorttext">Table or View Field Name</p>
+      "! This is the field name of table or view
+      field            TYPE c LENGTH 30,
+      "! <p class="shorttext">Operation Filter Value</p>
+      "! This is the operation filter value
+      operation        TYPE c LENGTH 10,
+      "! <p class="shorttext">Modeling Context Filter Low Value</p>
+      "! This is the low value of modeling context filter
+      value            TYPE string,
+      "! <p class="shorttext">Modeling Context Filter High Value</p>
+      "! This is the high value of modeling context filter
+      high_value       TYPE string,
+    END OF ty_modelling_context_filters,
+
+    "! <p class="shorttext">Parameter defaults</p>
+    "! These are the scenario parameter defaults
+    BEGIN OF ty_parameter_defaults,
+      "! <p class="shorttext">UUID</p>
+      "! This is the scenario UUID
+      guid    TYPE c LENGTH 36,
+      "! <p class="shorttext">Parameter Name</p>
+      "! This is the parameter name
+      name    TYPE c LENGTH 256,
+      "! <p class="shorttext">Parameter Context</p>
+      "! This is the parameter context
+      context TYPE c LENGTH 1000,
+      "! <p class="shorttext">Parameter Value</p>
+      "! This is the parameter value
+      value   TYPE string,
+    END OF ty_parameter_defaults,
+
+    "! <p class="shorttext">Parameter Signature</p>
+    "! These are the scenario parameters signature
+    BEGIN OF ty_parameters,
+      "! <p class="shorttext">Parameter Name</p>
+      "! This is the parameter name
+      name        TYPE c LENGTH 256,
+      "! <p class="shorttext">Parameter Type</p>
+      "! This is the parameter type
+      type        TYPE c LENGTH 32,
+      "! <p class="shorttext">Parameter Description</p>
+      "! This is the parameter description
+      description TYPE string,
+      "! <p class="shorttext">Storage</p>
+      "! This is the storage type
+      storage     TYPE c LENGTH 100,
```

Comment:
```suggestion
      storage_type     TYPE c LENGTH 100,
```

Reply:
If you want to usage `storage_type` I suggest also to adapt the title accordingly

Reply:
Done.